src/main.ts

```diff
@@ -103,14 +105,25 @@ const run = async (): Promise<void> => {
 
     const github = getOctokit(core.getInput('GITHUB_TOKEN'))
 
-    const jobs: ActionsListJobsForWorkflowRunJobs = await github.paginate(
+    let jobs: ActionsListJobsForWorkflowRunJobs = await github.paginate(
       github.rest.actions.listJobsForWorkflowRun.endpoint.merge({
         ...context.repo,
         run_id: githubRunId,
       })
     )
 
-    objectDebug('jobs', jobs)
+    jobs = jobs.map((job) => {
+      if (post && job.name.toLowerCase() === context.job) {
+        if (job.status === 'in_progress') {
```

Comment:
This is falsely assuming the job will complete. Any action running after this one might fail.

Reply:
This happens only during the **post** phase, if it has `in_progress` and it's already in the post step, it means that it completed, else it will have a failure status that is kept. We can try it out if you want :)

Reply:
@guerrerocarlos any other action declared after slacksync that runs in post will run and possibly fail after this runs.

Reply:
Maybe let's clarify in a meet today.